l1-contracts/contracts/common/Messaging.sol

```diff
@@ -137,12 +137,12 @@ struct BridgehubL2TransactionRequest {
 
 /// @dev The structure that contains the parameters for the message root
 /// @param chainId The chain id of the dependency chain
-/// @param blockOrBatchNumber The block number or the batch number where the message root was created
+/// @param blockNumber The block number where the message root was created
```

Comment:
Can you change this one back, and add a comment that for proof based interop it will only be block number. For commit based it will be batch as well.